AutomatedTesting/AutomationScripts/RunAtomScreenshotTests.bat

```diff
@@ -0,0 +1,3 @@
+pushd %~dp0
+../../python/python.cmd -s -B -m pytest -v --tb=short --show-capture=stdout -c ../../pytest.ini --build-directory ../../AutomatedTesting/build/bin/profile ../../AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Periodic_GPU.py --output-path ../../AutomatedTesting/build/Testing/LyTestTools/AutomatedTesting_Atom_TestSuite_Periodic_GPU --junitxml=../../AutomatedTesting/build/Testing/Pytest/AutomatedTesting_Atom_TestSuite_Periodic_GPU.xml
+popd
```

Comment:
Is this batch script needed, especially since it is Windows only and the registered test below can be run using `ctest -R AutomatedTesting::Atom_TestSuite_Periodic_GPU -C profile`

Reply:
I'm attempting to convince people to leave AtomSampleViewer behind by making it trivial to run these tests.
I plan to also add a bash script to do the same thing to make it easy on Linux (& Mac eventually)